srcpkgs/tap-plugins/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'tap-plugins'
+pkgname=tap-plugins
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+makedepends="ladspa-sdk"
+short_desc="Tom's LADSPA Plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+make_use_env=yes
+make_build_args="CC=${CC}"
```

Comment:
Sry I forgot, this does not work properly, it needs to be set inside of a function, putting it into `pre_build` does work

Reply:
Okay, fixed!